src/Native/build-native.sh

```diff
@@ -2,11 +2,13 @@
 
 usage()
 {
-    echo "Our parameters changed! The parameters: buildArch, buildType and HostOS"
-    echo "are passes by the Run Command Tool."
+    echo "Our parameters changed! The parameters: buildArch, buildType, buildOS, numProc"
+    echo "are passed by the Run Command Tool."
+    echo "If you plan to only run this script, be sure to pass those parameters in the stated order."
```

Comment:
It is just a number. But I don't know how to parse it. The Run Command Tool is passing it as the 4th option i.e. `x86 debug Linux 4` so I don't know how to add the parsing for the number.
By default it is 1 and if the user doesn't pass that 4th parameter then there are no issues.

Do you know of a better way to do this?